midp/gfx.js

```diff
@@ -686,26 +683,26 @@ var currentlyFocusedTextEditor;
     };
 
     Native["javax/microedition/lcdui/Graphics.getClipX.()I"] = function() {
-        return this.clipX1 - this.transX;
+        return this.clipX1;
     };
 
     Native["javax/microedition/lcdui/Graphics.getClipY.()I"] = function() {
-        return this.clipY1 - this.transY;
+        return this.clipY1;
     };
 
     Native["javax/microedition/lcdui/Graphics.getClipWidth.()I"] = function() {
-        return this.clipX2 - this.clipX1;
+        return this.clipX2;
```

Comment:
Why is this needed? Shouldn't the clipping area still be clipX2 - clipX1?

(The same applies to getClipHeight)